app/functions/EmbedFunctions/Program.cs

```diff
@@ -60,8 +67,10 @@
         services.AddSingleton<IEmbedService, AzureSearchEmbedService>(provider =>
         {
             var searchIndexName = Environment.GetEnvironmentVariable("AZURE_SEARCH_INDEX") ?? throw new ArgumentNullException("AZURE_SEARCH_INDEX is null");
-            var useAOAI = Environment.GetEnvironmentVariable("USE_AOAI")?.ToLower() == "true";
-            var useVision = Environment.GetEnvironmentVariable("USE_VISION")?.ToLower() == "true";
+            //var useAOAI = Environment.GetEnvironmentVariable("USE_AOAI")?.ToLower() == "true";
+            var useAOAI = true;
```

Comment:
looks like we are hard coding these values, they aren't keys so its okay. future we will just remove env variable and keep hard coded values.

same for next lines below